### PR TITLE
RC3 changelog.

### PR DESCRIPTION
**Pull request by vpetrovykh**

### CODE REVIEW

File: docs/changelog/1_0_rc3.rst [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+Now that the project has been re-initialized and data dump is
+restored, we need to restore the login credentials. Which presents a
+couple of options. We can restore the old credentials:
+
+.. code-block:: bash
+
+    $ edgedb instance reset-password my_instance --password-from-stdin
+
+Alternatively we can create new credentials. This is recommended if
```
Comment (elprans) [inline docs/changelog/1_0_rc3.rst:50]: Let's put the recommended option first and suggest `--password-from-stdin` iff `reset-password` was called on an instance previously and the password needs to be kept.

File: docs/changelog/1_0_rc3.rst [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+.. code-block:: bash
+
+    $ edgedb instance reset-password my_instance
+
+Alternatively we can restore the old credentials:
+
+.. code-block:: bash
+
+    $ edgedb instance reset-password my_instance --password-from-stdin
```
[the remark this answers is not in the document]
Comment (vpetrovykh) [reply docs/changelog/1_0_rc3.rst:88]: OK

File: docs/changelog/1_0_rc3.rst [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+restored, we need to restore the login credentials. Which presents a
+couple of options. If there's no code that uses the login credentials
+explicitly, but rather relies on the instance name only to access to
+the database, the recommended approach is to reset the password:
+
+.. code-block:: bash
+
+    $ edgedb instance reset-password my_instance
+
+Alternatively we can restore the old credentials:
```
[the remark this answers is not in the document]
Comment (vpetrovykh) [reply docs/changelog/1_0_rc3.rst:84]: That's why I added the paragraph about backing up credentials before even going into the upgrade workflows.

File: docs/changelog/1_0_rc3.rst [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+If you have an EdgeDB instance that's not part of a project, you can
+upgrade it to RC3 using the following steps:
+
+.. code-block:: bash
+
+    $ edgedb dump --all --format=dir upgrade_backup
+    $ edgedb instance destroy my_instance
+    $ edgedb instance create my_instance
+    $ edgedb restore --admin --all upgrade_backup
```
[the remark this answers is not in the document]
Comment (vpetrovykh) [reply docs/changelog/1_0_rc3.rst:72]: Yes! Good catch, I copied the lines and forgot the instance.

File: docs/changelog/1_0_rc3.rst [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+Instance upgrade
+----------------
+
+If you have an EdgeDB instance that's not part of a project, you can
+upgrade it to RC3 using the following steps:
+
+.. code-block:: bash
+
+    $ edgedb dump --all --format=dir upgrade_backup
```
[the remark this answers is not in the document]
Comment (vpetrovykh) [reply docs/changelog/1_0_rc3.rst:69]: I forgot the instance because I copied dump/restore from the project.

I've updated the backup directory as well.

**Review by 1st1 — APPROVED** [review]

**Pull request merged**